packages/ui-components/src/Containers/Sidebar/SidebarItem/index.module.css

```diff
@@ -4,16 +4,37 @@
   @apply font-regular
```

Comment:
I think the hover effect added to the Progression sidebar doesn’t align well with the overall site design.

<img width="366" height="185" alt="image" src="https://github.com/user-attachments/assets/1915201c-e69c-4439-a63f-51e2de4b56ed" />

<img width="366" height="101" alt="image" src="https://github.com/user-attachments/assets/c2db86fd-fc58-45e7-9bd7-f25d1ab4b5c5" />

IMO We could achieve a more consistent look by updating the span color to a lighter shade of the active SVG’s fill color like this;

<img width="366" height="185" alt="image" src="https://github.com/user-attachments/assets/87d58417-2ca9-4c29-bb9e-1d2ea1cf6cb4" />

Reply:
with lasted commit for me it's okay